addons/website_mail/views/website_mail.xml

```diff
@@ -69,7 +69,7 @@
             <ul class="media-list o_website_comments">
                 <t t-foreach="object.message_ids" t-as="message">
                     <t t-if="message.website_published or website.env.user.sudo(website.env.user.id).has_group('base.group_website_publisher')">
-                        <li class="media" t-if="message.message_type == 'comment'" t-att-id="'message-%s' % message.id">
+                        <li class="media" t-if="message.message_type != 'comment'" t-att-id="'message-%s' % message.id">
```

Comment:
You should check message.subtype_id.internal == False. But I think portal users cannot read subtype model. To test.